src/redux-flow/reducers/tic-tac-toe/action-creators.js

```diff
@@ -2,9 +2,17 @@ import {
   RESET_GAME,
   FILL_SQUARE,
   JOIN,
+  JOIN_ROOM,
```

Comment:
Separate the join room from the join to make is possible to recovery the room state before joining.